docs/Install.md

```diff
@@ -77,27 +77,39 @@ If you don't have an object storage provider, we recommend [MinIO](https://min.i
 
 ### Deploying Across Multiple Clusters
 
-By default, this tool will pull in data from the cluster in which it is running. The tool also supports collecting data across mulitple OpenShift clusters. In order to do this, we need to point the Pelorus instance at these other clusters.
+By default, this tool will pull in data from the cluster in which it is running. The tool also supports collecting data across mulitple OpenShift clusters. In order to do this, the thanos sidecar can be configured to read from a shared S3 bucket accross clusters. See [Pelorus Multi-Cluster Architecture](/page/Architecture.md) for details. You define exporters for the desired meterics in each of the clusters which metrics will be evaluated.  The main cluster's Grafana dashboard will display a combined view of the metrics collected in the shared S3 bucket via thanos.
```

Comment:
I think we're still missing an important piece here, which is implied but never actually stated, which is this: The way that you deploy Pelorus across multiple clusters is by deploying an independent Pelorus stack to each cluster you want to collect data from, and then tie them all together by pointing them to a single Object Storage backend. So to show that, I think we should show the entire `values.yaml` file example for each cluster we are deploying to, to show what is the same and what is different in each cluster.

Another idea is that we could use the deployment labels to designate the different types of deployments. For example:

```
deployment:
  labels:
    app.kubernetes.io/component: development
    app.kubernetes.io/name: pelorus
    app.kubernetes.io/version: v0.33.0
```

vs

```
deployment:
  labels:
    app.kubernetes.io/component: production
    app.kubernetes.io/name: pelorus
    app.kubernetes.io/version: v0.33.0
```

Reply:
I believe we should assume that the development cluster is the target here, so we should deploy the commit and failure exporters here.

Reply:
It looks as if Eric's concern is addressed.